helm/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I think we might need to add a persistent volume for the config/data so that if the pod is recreated we don't loose state

Reply:
We've added this now :)